javascript-loops-practice/includes-seven.js

```diff
@@ -0,0 +1,14 @@
+/* exported includesSeven */
+function includesSeven(array) {
+  var newArr = [];
+  for (var i = 0; i < array.length; i++) {
+    if (array[i] === 7) {
+      newArr.push(array[i]);
```

Comment:
Again, I would just `return true` here. There's no need to continue searching after you find a 7.